modules/pageDispaly.js

```diff
@@ -0,0 +1,28 @@
+const bookList = document.querySelector('.all-books');
+const addBookForm = document.querySelector('.add-book-box');
+const contact = document.querySelector('.contact-me-box');
+
+// const list = document.querySelector('.list');
+// const addBookLink = document.querySelector('.add-new');
+// const contactLink = document.querySelector('.contact');
+
```

Comment:
- [ ] Kindly remove this commented code to follow the JavaScript best practices 👍